interface/src/lib.rs

```diff
@@ -654,6 +659,19 @@ impl<'a> HostInterface for MockHost<'a> {
       static_context.nonce,
     )
   }
+
+  fn deploy(&mut self, code: Vec<u8>) -> Address {
+    // template_address := HASH(template_code)
+    let hash = blake3::hash(&code);
+    let hash_bytes = hash.as_bytes().as_slice();
+    let address = Address::try_from(&hash_bytes[..24]).unwrap();
```

Comment:
```suggestion
    let address = Address::try_from(&hash_bytes[..ADDRESS_LENGTH]).unwrap();
```

should update this in at least one more place in this file (where the hasher is used)